### PR TITLE
fix pager-duty and re-origanize

### PR DESCRIPTION
**Pull request by srikanthops**
Fix PD notifications and re-organize code

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -47,13 +45,34 @@ commands:
             elif [[ "${target}" == "stage" ]]; then
               url=api.lbcs.io
             else
-              url=api.lbvs.dev
+              url=api.lbcs.dev
```
Comment (edwardcallahan) [inline .circleci/config.yml:48]: oops

File: .circleci/config.yml [hunk 2]
```diff
@@ -135,165 +219,211 @@ jobs:
...
+          # mentions: "here"
           fail_only: true
-          only_for_branches: "master,cron-test"
+          only_for_branches: "master"
+      - slack/status:
+          channel: cloudstate-alerts
+          success_message: Shopping cart tests passed on Dev
+          failure_message: Shopping cart tests failed on Dev
+
+  # Prod-Var-Test
```
Comment (edwardcallahan) [inline .circleci/config.yml:354]: Why it is a '_var_' test? 
Validate API Resources?

File: .circleci/config.yml [hunk 3]
```diff
@@ -70,56 +89,121 @@ commands:
...
+                    return 1
                 fi
-            else
-                echo Unable to Shop "${SHOP_TEST_USER}" Cart not found!
-                false
-            fi
+                return 0
+            }
+            exitCode=0
+            # Try a three time after 5 and 10 seconds
```
Comment (edwardcallahan) [inline .circleci/config.yml:144]: noice

File: .circleci/config.yml [hunk 4]
```diff
@@ -135,165 +219,211 @@ jobs:
...
     steps:
       - cs_sc_test:
           cart_uri: "https://shrill-shadow-4688.us-east1.apps.cloudstate.com"
-      - slack/status:
+      - slack/status: # Failed cases
           channel: cloudstate-alerts
-          success_message: Shopping cart tests successful on prod
           failure_message: Shopping cart tests failed on prod
-          mentions: "@edc,@srikanth"
+          mentions: "here"
```
Comment (edwardcallahan) [inline .circleci/config.yml:290]: oh yeah

File: .circleci/config.yml (hunk 2 above)
Comment (edwardcallahan) [reply .circleci/config.yml:354]: If so, VAR test, no?

Comment (srikanthops) [reply .circleci/config.yml:354]: changed it to VAR

**Pull request merged**